diff --git tools/github_workflows/workflows_template.libsonnet tools/github_workflows/workflows_template.libsonnet
index 73bc06c..d52c884 100644
--- tools/github_workflows/workflows_template.libsonnet
+++ tools/github_workflows/workflows_template.libsonnet
@@ -52,7 +52,7 @@
               bazel_os: 'windows',
               cross_compile: false,
               lint: false,
-              os: 'windows-latest',
+              os: 'windows-2022',
               platform_name: 'windows_amd64',
               upload: false,
             },
