diff --git tools/github_workflows/workflows_template.libsonnet tools/github_workflows/workflows_template.libsonnet
index 53231de..d159c75 100644
--- tools/github_workflows/workflows_template.libsonnet
+++ tools/github_workflows/workflows_template.libsonnet
@@ -54,7 +54,7 @@
             {
               bazel_os: 'windows',
               cross_compile: false,
-              os: 'windows-latest',
+              os: 'windows-2022',
               platform_name: 'windows_amd64',
               upload: false,
             },
